src/main/java/example/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,60 @@
+package example.controller;
+
+import example.dto.cartItem.CartItemDto;
+import example.dto.cartItem.RequestCartItemDto;
+import example.dto.cartItem.RequestUpdateCartItemDto;
+import example.dto.shoppingCart.ShoppingRespondCartDto;
+import example.service.CartItemService;
+import example.service.ShoppingCartService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Positive;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
```suggestion
@RestController
@Validated
```
as you use Positive for Long id